models/coke_can/README.txt

```diff
@@ -0,0 +1 @@
+Only model.sdf has been edited everything else was left the same (not sure if all the files are even neccesary
```

Comment:
It's fine to leave the extra SDFs. I think the older format versions are long out of date, but they're small files.
Do remove the model tar though.